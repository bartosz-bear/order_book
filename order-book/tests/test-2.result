{"buyOrders": [], "sellOrders": [{"id": 1, "price": 100, "quantity": 100}]}
{"buyOrders": [], "sellOrders": [{"id": 1, "price": 100, "quantity": 100}, {"id": 2, "price": 100, "quantity": 100}]}
{"buyOrders": [], "sellOrders": [{"id": 1, "price": 100, "quantity": 100}, {"id": 2, "price": 100, "quantity": 100}, {"id": 3, "price": 100, "quantity": 100}]}
{"buyOrders": [], "sellOrders": [{"id": 3, "price": 100, "quantity": 100}, {"id": 2, "price": 100, "quantity": 100}]}
