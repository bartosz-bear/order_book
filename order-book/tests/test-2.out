{"sellOrders": [], "buyOrders": [{"id": 1, "price": 100, "quantity": 100}]}
{"buyOrderId": 1, "sellOrderId": 1, "price": 100,  "quantity": 50}
{"sellOrders": [], "buyOrders": [{"id": 1, "price": 100, "quantity": 50}]}
